crates/composefs-oci/src/skopeo.rs

```diff
@@ -10,12 +10,13 @@ use oci_spec::image::{Descriptor, ImageConfiguration, ImageManifest, MediaType};
 use rustix::process::geteuid;
 use tokio::{io::AsyncReadExt, sync::Semaphore};
 
-use composefs::{
-    fsverity::FsVerityHashValue, repository::Repository, splitstream::DigestMap, util::Sha256Digest,
-};
+use composefs::{fsverity::FsVerityHashValue, repository::Repository, util::Sha256Digest};
 
 use crate::{sha256_from_descriptor, sha256_from_digest, tar::split_async, ContentAndVerity};
 
+pub const TAR_LAYER_CONTENT_TYPE: u64 = 0x2a037edfcae1ffea;
```

Comment:
Can you add a comment for where these came from? I'm guessing random? If so just add a comment `// Random unique ID` ?

That said I wonder if it wouldn't be nicer to store (variable length) strings for this in the format? Maybe it could go all the way to literally suggested to be the mediaType from OCI (if applicable)

Reply:
Yes. These are random.

But, I'd rather avoid having variable length things in the header. That makes parsing it much more tricky.

Reply:
We could make it a real uuid tho

Reply:
I'm fine keeping as u64 too.

Reply:
I added some comments here